exporters/fluentd/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+# Copyright 2021, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+cmake_minimum_required(VERSION 3.12)
+
+project(opentelemetry-fluentd)
+
+add_definitions(-DHAVE_CONSOLE_LOG)
+add_definitions(-DENABLE_LOGS_PREVIEW)
+find_package(opentelemetry-cpp REQUIRED)
+find_package(nlohmann_json REQUIRED)
+find_package(CURL REQUIRED)
+
+if(COMMAND cmake_policy)
```

Comment:
Is this check needed as the required version 3.12 is quite new?

Reply:
Don't remember why it was added, have removed it now,